python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -191,6 +192,20 @@ def measure(self, measRecord, exposure):
         x1 = xc + dydtheta
         y1 = yc + dxdtheta
 
+        if not (0 <= x0 <= exposure.getBBox().endX and 0 <= x1 <= exposure.getBBox().endX
+                and 0 <= y0 <= exposure.getBBox().endY and 0 <= y1 <= exposure.getBBox().endY):
+
+            self.flagHandler.setValue(measRecord, self.EDGE.number, True)
+
+        else:
+
+            if exposure.mask[np.round(x0), np.round(y0)] and exposure.mask[np.round(x1), np.round(y1)]:
```

Comment:
An inline comment or two here would enhance clarity about what is happening

Reply:
Done